qemu/tests/nvdimm.py

```diff
@@ -158,6 +159,15 @@ def run(test, params, env):
         nvdimm_test.verify_nvdimm(vm, mems)
         error_context.context("Format and mount nvdimm in guest", logging.info)
         nvdimm_test.mount_nvdimm()
+        if params.get("nvml_test", "no") == "yes":
+            pkgs = params["depends_pkgs"].split()
+            if not utils_package.package_install(pkgs, nvdimm_test.session):
+                test.cancel("Install dependency packages failed")
+            nvdimm_test.run_guest_cmd(params["get_nvml"])
+            nvdimm_test.run_guest_cmd(params["compile_nvml"])
+            nvdimm_test.run_guest_cmd(params["config_nvml"])
+            nvdimm_test.run_guest_cmd(params["run_test"], timeout=3600)
```

Comment:
I doubt if the return status of “make check” can indicate test result. shall we add check for output of "make check"?

Reply:
The return status will tell the test result. E.g. it passes on ppc as no error found, and it fails on x86 with error output.

Reply:
If you are sure that,I have no problem.I run "make check" manually,and didn't hit failed,so I don't know what's the status if have failed cases.my test log:
make check-remote-quiet-conditional
make[3]: Entering directory '/root/nvml/src/test'
NOTE: Skipping remote tests because libfabric (version >= 1.4.2) is missing -- see src/librpmem/README for details
make[3]: Nothing to be done for 'check-remote-quiet-conditional'.
make[3]: Leaving directory '/root/nvml/src/test'
No failures.
make[2]: Leaving directory '/root/nvml/src/test'
make[1]: Leaving directory '/root/nvml/src'

Reply:
Yes, I'm sure about it. Thanks!